README.md

```diff
@@ -21,6 +181,9 @@ def deps do
 end
 ```
 
+## Documentation
```

Comment:
We use the README as the first page of hexdocs documentation. It can be confusing a section `documentation` inside the documentation. 🤔 

```suggestion
## API Documentation
```

Reply:
maybe `API spec` instead `documentation` would avoid it, what do you think?